src/test/java/org/opennms/minion/stests/SyslogTest.java

```diff
@@ -84,6 +84,12 @@ public void canReceiveSyslogMessages() throws Exception {
             pipe.println("config:update");
             // Install the syslog handler feature
             pipe.println("features:install opennms-syslogd-handler-default");
+            // Point the trap handler at the local ActiveMQ broker
+            pipe.println("config:edit org.opennms.netmgt.trapd.handler.default");
+            pipe.println("config:propset brokerUri tcp://127.0.0.1:61616");
+            pipe.println("config:update");
+            // Install the trap handler feature
+            pipe.println("features:install opennms-trapd-handler-default");
```

Comment:
True. I was thinking more about side-effects during the test itself of having both features installed. I'll combine the feature installation commands.